cmd/root_command.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"dp-cli/config"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	root *cobra.Command
+
+	r                    *rand.Rand
+	goPath               string
+	onsDigitalPath       string
+	hierarchyBuilderPath string
+	codeListScriptsPath  string
+	appVersion           string
+)
+
+func Load(cfg *config.Config) *cobra.Command {
+	s1 := rand.NewSource(time.Now().UnixNano())
+	r = rand.New(s1)
+
+	appVersion = "v0.0.1"
```

Comment:
Haven't done this myself either yet, @jessjenkins noticed that we haven't been incementing this version

Reply:
I've added a item to my task to compute this from git on build (if simple)

Reply:
@jessjenkins there is an example of how to do this in the [dp-healthcheck](https://github.com/ONSdigital/dp-healthcheck) readme.